TKGame/Weapons/WeaponSystem.cs

```diff
@@ -30,7 +30,7 @@ public WeaponSystem()
 
         public void Draw(SpriteBatch spriteBatch)
         {
-            spriteBatch.Draw(Art.WeaponTexture, weaponRectangle, Color.White);        
+            //spriteBatch.Draw(Art.WeaponTexture, weaponRectangle, Color.White);        
```

Comment:
What is the purpose of commenting this out?

Reply:
Because the game menu will be showing the texture so we don't need it to be done elsewhere (at least for now).